build/modern-loader.js

```diff
@@ -104,12 +104,12 @@ module.exports = async (id, content, onwarn=null) => {
     name: id,
     format: 'es',
     sourcemap: true,
-    treeshake: false,  // we want to cache the results
+    //treeshake: false,  // we want to cache the results
```

Comment:
Why not `treeshake: true,`

Reply:
I added a comment and fixed where this goes.

Reply:
In dev we just use Rollup to rewrite imports and convert to CJS—we only ever go from one file to one file, it's just a transpiler. We don't want it to do anything more than that.